### PR TITLE
Wrong content on favorite cta depending on viewport

### PR DESCRIPTION
**Pull request by wazolab**

### CODE REVIEW

File: locales/es-ES.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default defineI18nLocale(() => {
       backToMenuFavorites: 'Ver el menú',
       backToMenuExplorer: 'Buscar par categorías',
       backToMenuFavoritesMobile: 'Salir de los favoritos',
-      backToMenuExplorerMobile: 'Buscar par categorías',
+      backToMenuExplorerMobile: 'Salir \'Cercano\'',
```
Comment (frodrigo) [inline locales/es-ES.ts:82]:
```suggestion
      backToMenuExplorerMobile: 'Salir «Cercano»',
```

File: locales/en-GB.ts [hunk 2]
```diff
@@ -77,9 +77,9 @@ export default defineI18nLocale(() => {
       clearAllCategories: 'Hide all categories',
       back: 'Back',
       backToMenuFavorites: 'Show menu',
-      backToMenuExplorer: 'Search by Categories',
-      backToMenuFavoritesMobile: 'Exit Favorites',
-      backToMenuExplorerMobile: 'Search by Categories',
+      backToMenuExplorer: 'Search by categories',
+      backToMenuFavoritesMobile: 'Exit favorites',
+      backToMenuExplorerMobile: 'Exit \'Nearby\'',
```
Comment (frodrigo) [inline locales/en-GB.ts:82]:
```suggestion
      backToMenuExplorerMobile: 'Exit "Nearby"',
```

File: locales/fr-FR.ts [hunk 3]
```diff
@@ -80,7 +80,7 @@ export default defineI18nLocale(() => {
       backToMenuFavorites: 'Afficher le menu',
       backToMenuExplorer: 'Rechercher par catégories',
       backToMenuFavoritesMobile: 'Quitter les favoris',
-      backToMenuExplorerMobile: 'Rechercher par catégories',
+      backToMenuExplorerMobile: 'Quitter \'À proximité\'',
```
Comment (frodrigo) [inline locales/fr-FR.ts:83]:
```suggestion
      backToMenuExplorerMobile: 'Quitter « À proximité »',
```

File: locales/es-ES.ts (hunk 1 above)
Comment (wazolab) [reply locales/es-ES.ts:82]: no spaces before/after « » ?

File: locales/en-GB.ts (hunk 2 above)
Comment (wazolab) [reply locales/en-GB.ts:82]: why not using « » ? it's language specific ?

Comment (frodrigo) [reply locales/en-GB.ts:82]: No, only french typo do that.

File: locales/es-ES.ts (hunk 1 above)
Comment (frodrigo) [reply locales/es-ES.ts:82]: No, only french typo do that.

File: locales/en-GB.ts (hunk 2 above)
Comment (frodrigo) [reply locales/en-GB.ts:82]: Yes it is by language.

**Pull request merged**